model/common/src/icon4py/model/common/grid/icon.py

```diff
@@ -213,7 +213,10 @@ def start_index(self, domain: h_grid.Domain) -> gtx.int32:
         if domain.local:
             # special treatment because this value is not set properly in the underlying data.
             return gtx.int32(0)
-        return gtx.int32(self._start_indices[domain.dim][domain()])
+        if hasattr(self._end_indices[domain.dim][domain()], "get"):
```

Comment:
this is removed again in #685 which is open against this PR.

Reply:
Perfect.